### PR TITLE
Hover texts in masthead fixed and changed

### PR DESCRIPTION
**Pull request by yodem**
Fixes: #949

The texts on the masthead changed and designed like patternfly 4 tooltips.
admin_Hovering
Notifications_Hover
Refresh_Hovering

### CODE REVIEW

**Comment by sjd78** [comment]
> I think the `OverlayTrigger` should directly wrap the `a` tag in each case to be consistent and make sure it doesn't stay there with About/Options dropdown menu.

Location of the Tooltips look good now.

> Generally looks good, but now we have a lot of cases where we using `Tooltip` with `OverlayTrigger`, what means a lot of similar code. So I think it will be better to create component with that code and use it instead.

@yodem - You can create a component like this, drop it in `src/components`, and use it as Bohdan suggests...
```jsx
const OverlayTooltip = ({ id, tooltip, placement = 'bottom', children, ...rest }) =>
  <OverlayTrigger
    overlay={
      <Tooltip id={id}>{tooltip}</Tooltip>
    }
    placement={placement}
    {...rest}
  >
    { children }
  </OverlayTrigger>
```

**Comment by sgratch** [comment]
@yodem @lwrigh Regarding hover text over user icon: do we want it to include only "admin" or also the provider, as appears on webAdmin:
Screenshot from 2019-08-25 15-04-23

**Comment by yodem** [comment]
> @yodem @lwrigh Regarding hover text over user icon: do we want it to include only "admin" or also the provider, as appears on webAdmin:
> Screenshot from 2019-08-25 15-04-23

I think that admin is enough, adding the provider make it more complex and I think its unnecessary.

**Comment by yodem** [comment]
> > I think the `OverlayTrigger` should directly wrap the `a` tag in each case to be consistent and make sure it doesn't stay there with About/Options dropdown menu.
> 
> Location of the Tooltips look good now.
> 
> > Generally looks good, but now we have a lot of cases where we using `Tooltip` with `OverlayTrigger`, what means a lot of similar code. So I think it will be better to create component with that code and use it instead.
> 
> @yodem - You can create a component like this, drop it in `src/components`, and use it as Bohdan suggests...
> 
> ```js-jsx
> const OverlayTooltip = ({ id, tooltip, placement = 'bottom', children, ...rest }) =>
>   <OverlayTrigger
>     overlay={
>       <Tooltip id={id}>{tooltip}</Tooltip>
>     }
>     placement={placement}
>     {...rest}
>   >
>     { children }
>   </OverlayTrigger>
> ```

see #1098

**Comment by lwrigh** [comment]
I would tend to agree with @yodem with just having "Admin" as the hover text. Having the provider also seems like overkill.

File: src/components/VmsPageHeader/UserMenu.js [hunk 1]
```diff
@@ -13,9 +14,11 @@ const UserMenu = ({ config, onLogout }) => {
   const idPrefix = 'usermenu'
   return (
     <li className='dropdown'>
-      <a className='dropdown-toggle nav-item-iconic' href='#' data-toggle='dropdown' title={config.getIn(['user', 'name'])} id={`${idPrefix}-user`}>
-        <i className='pficon pficon-user' /><span className='caret' />
-      </a>
+      <OverlayTrigger overlay={<Tooltip>{config.getIn(['user', 'name'])}</Tooltip>} placement='bottom'>
```
Comment (sjd78) [inline src/components/VmsPageHeader/UserMenu.js]: This should be updated to use `OverlayTooltip` like the others.

**Comment by isaranova** [comment]
LGTM

**Pull request merged**